packages/core/src/execution-engine/scheduled-task-manager.ts

```diff
@@ -49,10 +49,10 @@ export class ScheduledTaskManager {
 
 	registerCron(ctx: CronContext, onTick: () => void) {
 		const { workflowId, timezone, nodeId, expression, recurrence } = ctx;
-
+		const cronDescription = typeof expression === 'string' ? expression : expression.toString();
```

Comment:
Might as well call it always if we've to check anyway?

```suggestion
		const cronDescription = expression.toString();
```